src/NServiceBus.Core/Transports/PushRuntimeSettings.cs

```diff
@@ -1,16 +1,25 @@
 namespace NServiceBus.Transport
 {
+    using System;
+
     /// <summary>
-    /// Contains the limits for how messages should be dequeued.
+    /// Controls how the message pump should behave.
     /// </summary>
     public class PushRuntimeSettings
     {
         /// <summary>
-        /// Restricts the concurrency.
+        /// Constructs the settings.
         /// </summary>
-        /// <param name="maxConcurrency">The max value to enforce.</param>
-        public PushRuntimeSettings(int maxConcurrency = 100)
+        /// <param name="maxConcurrency">The maximum concurrency to allow. 0 allows NServiceBus to pick the a suitable default.</param>
```

Comment:
This sentence should probably be "Zero" instead of "0", or reworded to have have "0" starting the sentence.

Also, there's a typo: "allows NServiceBus to pick ~~the~~ a suitable default"